backend-project/small_eod/generic/tests.py

```diff
@@ -71,30 +70,28 @@ def get_create_data(self):
         return data
 
 
-class FactoryTestCase(TestCase):
+class FactoryCreateObjectsMixin:
 
-    FACTORIES: List[Tuple[Type[Model], Type[DjangoModelFactory]]] = ()
-    FACTORIES_MANY: int = 10
+    FACTORY: Tuple[Type[Model], Type[DjangoModelFactory]] = ()
+    FACTORY_MANY_COUNT: int = 10
 
-    @tag(tags.FACTORY_SINGLE)
+    @tag('FactoryCreateObjectsMixin')
     def test_factories_simple(self):
-        for e in self.FACTORIES:
-            model, factory = e[0], e[1]
+        model, factory = self.FACTORY[0], self.FACTORY[1]
+        factory()
+        self.assertEqual(
+            1,
+            model.objects.all().count(),
+            msg=f"Failed simple factory test, model: {model} with factory: {factory}"
```

Comment:
Skoro już nie masz pętli to informacje te wynikają z nazwy klasy, więc nie jest to konieczne (zostaw). W momencie, gdy robisz assercje w pętli to taki komunikat jest konieczny, bo po linijce nie ustalisz ile przejść pętli się już udało.